tests/conftest.py

```diff
@@ -300,15 +300,44 @@ def exported_kubeconfig(unprivileged_secret, kubeconfig_export_path):
 
 
 @pytest.fixture(scope="session")
-def admin_client():
+def local_admin_client(kubeconfig_export_path):  # ---> local_admin_client   vs    remove_admin_client
```

Comment:
I do not think we need this name change.
`local` vs `remote` is not meaningful, what is the terminology used for ocp2ocp migration?